eds/blocks/hero/hero.js

```diff
@@ -22,4 +24,23 @@ export default function decorate(block) {
 
   if (videoElement) videoElement.append(videoSrc);
   if (videoAssets) block.prepend(videoElement);
+
+  const contentDiv = block.querySelector('div:last-child');
+  const contentChildDiv = contentDiv.querySelector('div');
+  const children = [...contentChildDiv.children];
+
+  const heroLeft = domEl('div', { class: 'hero-left' });
+  const heroContentWrapper = domEl('div', { class: 'hero-content-wrapper' });
+  const heroRight = domEl('div', { class: 'hero-right' });
```

Comment:
Prefer semantic names over direction/placement names. `hero-content` `hero-image` rather than `-left | -right'. In RTL lang, 'left' is 'right'. It also then more accurately describes what should be in the container.